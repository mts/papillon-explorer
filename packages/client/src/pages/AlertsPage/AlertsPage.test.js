import React from 'react';
import alertsPageDefault from './AlertsPage.render'

jest.mock('react-router-dom', () => {
  return {
    Link: () => <div />
  }
});

describe('<AlertsPage />', () => {
  describe('Snaphot', () => {
    test('must match alertsPageDefault', () => {
      expect(global.renderToJSON(alertsPageDefault)).toMatchSnapshot()
    })
  })
})
