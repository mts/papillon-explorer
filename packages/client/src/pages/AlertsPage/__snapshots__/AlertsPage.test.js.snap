// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlertsPage /> Snaphot must match alertsPageDefault 1`] = `
<div>
  <h2>
    Alerts Page
  </h2>
  <div>
    Under construction ðŸ™‚
    <div />
  </div>
</div>
`;
